tests/neg/i12211.scala

```diff
@@ -7,7 +7,7 @@ val y: { def f(x: String): String } = x  // error: type mismatch (different sign
 class Sink[A] { def put(x: A): Unit = {} }
 class Sink1[A] extends Sink[A] { def put(x: "123") = ??? }
 
-@main def Test =
+@main def Test(args: String*) =
```

Comment:
We will be able to revert all these once #14204 is merged.

Reply:
Now we can rebase and revert those changes. Now, the testing framework does not add the string argument.